pkg/daemon/iptables/remote.go

```diff
@@ -0,0 +1,41 @@
+package iptables
+
+import (
+	"fmt"
+	"net"
+)
+
+func (mgr *Manager) RecordRemoteNodeIP(nodeIP net.IP) {
+	mgr.remoteNodeIPList = append(mgr.remoteNodeIPList, nodeIP)
+}
+
+func (mgr *Manager) RecordRemoteSubnet(cluster string, subnetCidr *net.IPNet, isOverlay bool) error {
+	if err := mgr.remoteSubnetTracker.Track(subnetCidr.String(), cluster); err != nil {
+		return err
+	}
+
+	if isOverlay {
+		mgr.remoteOverlaySubnet = append(mgr.remoteOverlaySubnet, subnetCidr)
+	} else {
+		mgr.remoteUnderlaySubnet = append(mgr.remoteUnderlaySubnet, subnetCidr)
+	}
+
+	mgr.remoteCidr.Add(subnetCidr.String())
+	return nil
+}
+
+func (mgr *Manager) configureRemote() (bool, error) {
+	if len(mgr.remoteOverlaySubnet) == 0 && len(mgr.remoteUnderlaySubnet) == 0 {
+		return false, nil
+	}
+
+	if err := mgr.remoteSubnetTracker.Conflict(); err != nil {
```

Comment:
It seems mananger has check the confliction of remote subnet, we don't need to check again.